Software/DumptruckUltra-MCU/hw/i2c/test/test_i2cbus.cpp

```diff
@@ -0,0 +1,49 @@
+#include "hw/i2c/i2cBusManager.hpp"
+#include "hw/proc/proc_setup.hpp"
+
+int main() {
+    Hardware::Processor::setupProcessor();
+
+    // Setup I2C
+    I2CBusManager::i2cPin_t i2cPins = {.sda = P5_1, .scl = P5_0};
+    I2CBusManager i2cBus(&i2cPins);
+
+    uint8_t value = 0x00;
+    i2cBus.i2cWriteReg(0x20, 0x03, &value, 1);
+
+    i2cBus.i2cWriteReg(0x20, 0x02, &value, 1);
+
+    value = 0x01;
+    i2cBus.i2cWriteReg(0x20, 0x01, &value, 1);
+    cyhal_system_delay_ms(500);
```

Comment:
I would comment somewhere about what these registers are so we don't have to look at the datasheet if we look at this again.